examples_gbs/run_tutorial_similarity.py

```diff
@@ -0,0 +1,278 @@
+# we need to convert the adjacency matrices into NetworkX Graphs:
+
+import networkx as nx
+import plotly
+
+plot_mutag_0 = plot.plot_graph(nx.Graph(m0_a))
+plot_mutag_1 = plot.plot_graph(nx.Graph(m1_a))
+plot_mutag_2 = plot.plot_graph(nx.Graph(m2_a))
+plot_mutag_3 = plot.plot_graph(nx.Graph(m3_a))
+
+plotly.offline.plot(plot_mutag_0, filename="MUTAG_0.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_0.html
+
+plotly.offline.plot(plot_mutag_1, filename="MUTAG_1.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_1.html
+
+plotly.offline.plot(plot_mutag_2, filename="MUTAG_2.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_2.html
+
+plotly.offline.plot(plot_mutag_3, filename="MUTAG_3.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/MUTAG_3.html
+#
+# By visual inspection, we see that the graphs of ``m1_a`` and ``m2_a`` look very similar. In fact,
+# it turns out that they are *isomorphic* to each other, which means that the graphs can be made
+# identical by permuting their node labels.
+#
+# Samples from these graphs can be accessed by indexing:
+
+print(m0[0])
+
+##############################################################################
+# Creating a feature vector
+# -------------------------
+#
+# Following :cite:`schuld2019quantum`, we can create a *feature vector* to describe each graph.
+# These feature vectors contain information about the graphs and can be viewed as a mapping to a
+# high dimensional feature space, a technique often used in machine learning that allows us to
+# find ways of separating points in the space for classification.
+#
+# The feature vector of a graph can be composed in a variety of ways. One approach is to built it
+# using statistics from a GBS device configured to sample from the graph, as we now discuss.
+#
+# We begin by defining the concept of an *orbit*, which contains all the GBS samples that are
+# equivalent under permutation of the modes. A sample can be converted to its corresponding orbit
+# using the :func:`~.sample_to_orbit` function. For example, the first sample of ``m0`` is ``[0,
+# 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0]`` and has orbit:
+
+print(similarity.sample_to_orbit(m0[0]))
+
+##############################################################################
+# Other samples can be randomly generated from the ``[1, 1]`` orbit using:
+
+print(similarity.orbit_to_sample([1, 1], modes=m0.modes))
+
+##############################################################################
+# Orbits provide a useful way to coarse grain the samples from GBS into outcomes that are

```

Comment:
```suggestion
# Orbits provide a useful way to coarse-grain the samples from GBS into outcomes that are
```

Reply:
Not 100% sure, but I think this use of coarse-grain as a verb should be hyphenated